From 865c02f71483195c8f27ec94689833eb00049df2 Mon Sep 17 00:00:00 2001
From: James Slagle <jslagle@redhat.com>
Date: Mon, 3 Feb 2014 13:29:28 -0500
Subject: [PATCH 02/14] Add create-dir service for neutron

The service will ensure /var/run/neutron is created and neutron owned,
which is required by the neutron daemon.

Change-Id: I833b325e24da872e7290fac3d06c04a92980e4c7
---
 elements/neutron/install.d/neutron-package-install/76-neutron | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/elements/neutron/install.d/neutron-package-install/76-neutron b/elements/neutron/install.d/neutron-package-install/76-neutron
index 2eb8ef5..ce85a3f 100755
--- a/elements/neutron/install.d/neutron-package-install/76-neutron
+++ b/elements/neutron/install.d/neutron-package-install/76-neutron
@@ -6,10 +6,7 @@ install-packages openstack-neutron openstack-neutron-ml2
 
 ln -s /etc/neutron/plugins/ml2/ml2_conf.ini /etc/neutron/plugin.ini
 
-# Needed because the packaged install uses /var/lib/neutron, but we configure
-# it to be /var/run/neutron.
-mkdir /var/run/neutron
-chown neutron.neutron /var/run/neutron
+os-svc-daemon -n neutron-server -u neutron -c noop -d
 
 # Packaged neutron is providing a requires.txt which breaks dependencies at
 # runtime. See:
-- 
1.8.5.3

